css/pages/index.css

```diff
@@ -4,6 +4,12 @@
   box-sizing: border-box;
 }
 
+html,
+body {
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
Can we get some alternative to fix the horizontal scroll like fixing the CSS of elements which are causing horizontal scroll ??

Reply:
Couldn't figure out the css of element, so have assigned it on body because horizontal scroll for each screen div didnt made sense. created class for body tag and assigned overflow attribute there

Reply:
I am saying that probably some extra margin/padding or negative margin/padding is causing the horizontal scroll. Can you please figure out the root cause of this issue ??